python/snewpy/models/base.py

```diff
@@ -161,6 +162,33 @@ def get_transformed_spectra(self, t, E, flavor_xform):
 
         return transformed_spectra   
 
+    def get_flux (self, t, E, distance, flavor_xform=NoTransformation(), **kwargs):
+        """Get neutrino flux through 1cm^2 surface at the given distance
+
+        Parameters
+        ----------
+        t : astropy.Quantity
+            Time to evaluate the neutrino spectra.
+        E : astropy.Quantity or ndarray of astropy.Quantity
+            Energies to evaluate the the neutrino spectra.
+        distance : astropy.Quantity or float (in kpc)
+            Distance from supernova to the 
+        flavor_xform : FlavorTransformation
+            An instance from the flavor_transformation module.
+
+        Returns
+        -------
+        dict
+            Dictionary of neutrino fluxes in [neutrinos/(cm^2*MeV*s)], 
```

Comment:
```suggestion
            Dictionary of neutrino fluxes in [neutrinos/(cm^2*erg*s)], 
```
`get_transformed_spectra` returns `<Quantity 1.234e+56 1 / (erg s)>`